public/app/features/provisioning/FileStatusPage.tsx

```diff
@@ -162,8 +143,9 @@ function ResourceView({ wrap, repo, repoRef, tab }: Props) {
               )}
             </AutoSizer>
           </div>
-          <Stack direction={'row'}>
+          <Stack direction={'row'} alignItems="flex-end" justifyContent="end" gap={1}>
             <Button
```

Comment:
```suggestion
          <Stack alignItems="flex-end" justifyContent="end">
            <Button
```
'row' and gap={1} are default.